ui/lib/replication/addon/templates/components/replication-summary.hbs

```diff
@@ -342,17 +342,41 @@
   {{#if (eq replicationAttrs.mode 'initializing')}}
     The cluster is initializing replication. This may take some time.
   {{else}}
-    {{info-table-row label="Mode" value=replicationAttrs.mode}}
-    {{info-table-row label="Replication set" value=replicationAttrs.clusterId}}
-    {{info-table-row label="Secondary ID" value=replicationAttrs.secondaryId}}
-    {{info-table-row label="State" value=replicationAttrs.state}}
-    {{info-table-row label="Primary cluster address" value=replicationAttrs.primaryClusterAddr}}
-    {{info-table-row label="Replication state" value=replicationAttrs.replicationState}}
-    {{info-table-row label="Last WAL entry" value=replicationAttrs.lastWAL}}
-    {{info-table-row label="Last Remote WAL entry" value=replicationAttrs.lastRemoteWAL}}
-    {{info-table-row label="Merkle root index" value=replicationAttrs.merkleRoot}}
-    {{#if replicationAttrs.syncProgress}}
-      {{info-table-row label="Sync progress" value=(concat replicationAttrs.syncProgress.progress '/' replicationAttrs.syncProgress.total)}}
-    {{/if}}
+    <div class="replication">
+      <ReplicationToggle />
+      <div class="selectable-card-container">
+        {{!-- TODO: there are 2 attributes for both state and lastWAL. which one do I use? --}}
+        <ReplicationPrimaryCard
+          @title='State'
+          @description='Updated every ten seconds.'
+          @metric={{replicationAttrs.state}} />
+        <ReplicationPrimaryCard
+          @title='Last WAL entry'
+          @description='Index of last Write Ahead Logs entry written on local storage.'
+          @metric={{replicationAttrs.lastWAL}}
+          />
+
+        {{!-- TODO: move this into its own component --}}
+        <div class="selectable-card is-rounded secondaries">
+          <div class="level">
+          <code class="card-title title is-5">Known Secondaries</code>
+            <ToolbarLink @params={{array "vault.cluster.settings.auth.configure" model.id}} @data-test-manage-link="true">
+              Manage
+            </ToolbarLink>
+          </div>
+        <div>
+
+            {{!-- TODO: pass in model or model.dr to populate with real data --}}
+            <ReplicationTable />
+          </div>
+        </div>
+      </div>
+      {{!-- TODO: improve this API to pass in model.dr, or rename metric_1 to merkleRoot etc --}}
+      <ReplicationTableRows
+          @metric_1={{if model.dr.merkleRoot model.dr.merkleRoot 'unknown' }}
```

Comment:
@Monkeychip since this table renders the same metrics (merkle root, mode, replication set, and sync progress) across all dashboards, i am included to either explicitly rename each metric (eg. `@metric_1` --> `@mode`) or simply pass in the `model.dr` into the component. this way it'd be clearer that we only expect those metrics. what do you think?

Reply:
I _think_ this view get's used on the Performance pages as well. Which means, depending on the page we'd want access to model.performance.  So maybe explicit rename each metric?